framework/validators/UniqueValidator.php

```diff
@@ -133,7 +145,24 @@ public function validateAttribute($model, $attribute)
         }
 
         if ($exists) {
-            $this->addError($model, $attribute, $this->message);
+            if (is_array($targetAttribute)) {
+                $attributeCombo = [];
```

Comment:
Hm, can't get rid of word "combo" everywhere in the same way: there are also message placeholders {attributeCombo} and {valueCombo}. Saying {attributeComposition} and {valueComposition} sounds wrong. Besides, `comboNotUnique` is consistent with other validators names like `tooBig`, `wrongMimeType`, `ipv4NotAllowed`, etc. And `compositeMessage` would break the consistency. So I will name the method `addComboNotUniqueError`.